components/basketball/players/PlayerListOfTeam.tsx

```diff
@@ -4,15 +4,18 @@ import { Link, Image } from "@nextui-org/react";
 interface PlayerListOfTeamProp {
   seasonTeamPlayers: BbSeasonTeamPlayer[];
   params: any;
+  playerId: String;
```

Comment:
prefer to be number
and let parent handle `String` from `number`

Reply:
Fixed.